### ISSUE TITLE
CycleGAN visdom 4th image in each row is switched

### ISSUE THREAD

**Issue by Oewyn** [junyanz/pytorch-CycleGAN-and-pix2pix#517]
It seems like in visdom the 4th image in each row (labeled idt_A and idt_B respectively) is reversed. the top right most image is the identity loss for (Real_B - Rec_B) and the bottom right image is the identity loss for (Real_A - Rec_A).

**Comment by junyanz** [comment]
Yes. I guess people may have different preferences regarding the ordering in visualization. You can change the order by modifying this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/cycle_gan_model.py#L60).

**Comment by Oewyn** [comment]
Let me include an image to make it a little more clear.

![visdom](https://user-images.githubusercontent.com/24667483/52434657-c0e99180-2ac4-11e9-8df0-fb4f87396c26.jpg)

Modifying those 2 lines will switch right most images? I thought those lines would only impact the labels.

I believe the label ordering makes sense, i think it makes more sense to change the image ordering as shown in in the picture so that all images derived from the same real image are on the same row.

**Comment by Oewyn** [comment]
I did verify that your suggested fix does indeed switch the images, it also switches the labels, which isn't so bad.
